packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -50,12 +50,9 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ReinitializableBase
     /// @notice The L2 gas limit set when eth is deposited using the receive() function.
     uint64 internal constant RECEIVE_DEFAULT_GAS_LIMIT = 100_000;
 
-    /// @notice The L2 gas limit for system deposit transactions that are initiated from L1.
-    uint32 internal constant SYSTEM_DEPOSIT_GAS_LIMIT = 200_000;
```

Comment:
This was just an unused var? When did we stop using it?

Reply:
It was introduced by CGT and it got missed when CGT was removed